evm/src/cpu/decode.rs

```diff
@@ -1,3 +1,9 @@
+// We want to access `All`, `User`, and `Kernel` without having to go through `Availability`. Rust
+// wants constants to be all upper case, but these aren't really constants, just shortcuts. Also,
+// this syntax is consistent with the Rust std (think `None` and `Some`, which are not all upper
+// case).
+#![allow(non_upper_case_globals)]
+
```

Comment:
Totally unimportant but you can get the same behavior by adding the line
```rust
use Availability::{All, Kernel, User};
```

Reply:
Oh, that is much nicer! Thanks :)